packages.yml

```diff
@@ -1,7 +1,7 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: 1.0.0
+    version: 1.3.0
   - package: elementary-data/elementary
-    version: 0.15.2
+    version: 0.16.2
   - package: calogica/dbt_expectations
```

Comment:
Because of the sqlfluff error, I run dbt deps again and it returned that "dbt-utils and dbt-elementry" has new versions now, so for being at a safe side I upgraded both the version